state_of_html.yml

```
@@ -84,56 +84,56 @@ translations:
   # forms
 
   - key: features.form_controls
-    t: Form Controls
+    t: 表單控制
   - key: features.form_controls.question
-    t: Which form controls have you used?
+    t: 使用過哪些表單控制？
 
   - key: features.form_input_types
-    t: Input Types
+    t: 輸入類型
   - key: features.form_input_types.question
-    t: Which input types have you used?
+    t: 使用過哪些輸入類型？
   - key: options.form_input_types.na
-    t: <span aria-hidden="true">🚫</span> None of the above
+    t: <span aria-hidden="true">🚫</span> 以上皆非
 
   - key: features.form_validation_features
-    t: Form Validation
+    t: 表單驗證
   - key: features.form_validation_features.question
-    t: Which of the following form validation features have you used?
+    t: 使用過以下哪些表單驗證功能？
   - key: options.form_validation_features.na
-    t: <span aria-hidden="true">🚫</span> None of the above
+    t: <span aria-hidden="true">🚫</span> 以上皆非
 
   - key: features.forms_pain_points
-    t: Form Pain Points
+    t: 表單的痛點
   - key: features.forms_pain_points.question
-    t: What are your pain points around HTML forms?
+    t: 使用 HTML 表單有哪些痛點？
   - key: features.forms_pain_points.description
     t: |
-      Think of the various form controls (e.g. `<input>`, `<textarea>`, `<select>`, `<button>` etc.),
-      behaviors (e.g. validation, autocomplete), handling forms via JS, editing APIs (e.g. `contentEditable`) etc.
-      Anything that bothers you is fair game — browser support, missing features, styling, etc.
+      考慮各種表單選項（例如
+      ：`<input>`, `<textarea>`, `<select>`, `<button>` 等等。）、行為（例如：驗證、自動完成）、透過 JS 處理表單、編輯 APIs（例如：`contentEditable`）等等。
+      任何困擾你的事情都是公平的——瀏覽器支援、缺少的功能、樣式等等。
   - key: features.forms_pain_points.placeholder
     t: |
-      Pain point #{index}…
+      痛點 #{index}…
 
   # interactivity
 
   - key: features.dom_attribute_features
-    t: DOM Attribute-related methods
+    t: DOM 屬性相關方法
   - key: features.dom_attribute_features.question
-    t: Which of the following DOM attribute-related methods have you used?
+    t: 使用過以下哪些與 DOM 屬性相關的方法？
 
   - key: features.dom_html_features
-    t: HTML String-handling Features
+    t: HTML 字串處理功能
   - key: features.dom_html_features.question
-    t: Which of the following DOM HTML string-handling features have you used?
+    t: 使用過以下哪些 DOM HTML 字串處理功能？
 
   - key: features.dom_methods
-    t: Element reparenting DOM Methods
+    t: Element reparenting DOM Methods元素重新調整的 DOM 方法
```

```suggestion
    t: 移動的 DOM 到其他元件下的方法
```

> 部分我覺得保留原文比較適當所以沒翻譯 

不知道這是你說的嗎？因為看起來是重複的內容

---

這邊不是，是我忘了刪掉原文😅